code/modules/mob/living/carbon/xenomorph/death.dm

```diff
@@ -118,6 +118,17 @@
 
 	..(1)
 
+/mob/living/carbon/Xenomorph/Hunter/gib()
+
+	var/obj/effect/decal/remains/xeno/remains = new(get_turf(src))
+	remains.icon = icon
+	remains.pixel_x = pixel_x //For 2x2.
+
+	remains.icon_state = "Hunter Gibs"
+
+	check_blood_splash(35, BURN, 65, 2) //Some testing numbers. 35 burn, 65 chance.
+
+	..(1)
```

Comment:
```suggestion
	return ..()
```
There is nothing that takes a parameter in the parent procs.

Reply:
I took it as I found it